src/screens/Signup/index.tsx

```diff
@@ -112,6 +112,14 @@ export function Signup({onPressBack}: {onPressBack: () => void}) {
       }
     }
 
+    logEvent('signup:nextPressed', {
+      activeStep: state.activeStep,
+      phoneVerificationRequired:
+        state.activeStep === SignupStep.HANDLE
+          ? Boolean(state.serviceDescription?.phoneVerificationRequired)
+          : undefined,
```

Comment:
It should be defined on the first step too right? Could always add it here in that case. I suppose if we're only checking for this value on `activeStep: 1` then it's same difference though.

Reply:
Oh yea, I just added it to this step so that we can explicitly see if it's undefined here for some reason. Seems unlikely but idk.

Reply:
just made it always show up, can just determine which screen by `activeStep` like you said 👍